app/src/main/res/values/strings.xml

```diff
@@ -1,15 +1,14 @@
 <resources>
     <string name="app_name">Webmaker</string>
 
-
     <string name="title_section1">Discover</string>
     <string name="title_section2">Make</string>
-
-    <string name="hello_world">Hello world!</string>
-
-    <string name="action_settings">Settings</string>
-    <string name="title_activity_map">Map</string>
-    <string name="title_activity_project">Project</string>
-    <string name="title_activity_editor">Editor</string>
-    <string name="title_activity_tinker">Tinker</string>
+    <string name="title_activity_map">Project</string>
```

Comment:
this is setting off my "clean refactor" senses. Either that activity name, or its content string, need to be brought in line,